doordashcon: retain at rank I (#54)
polkadot-fellows/Evaluations: evidence/doordashcon/0002-doordashcon-08-2024.md

# Evidence-0002: Retention at Rank I

|                 |                                                                                             |
| --------------- | ------------------------------------------------------------------------------------------- |
| **Report Date** | Date of submission (2024/08/21)                                                             |
| **Submitted by**| Jesse Chejieh                                                 |

## Member details

- Matrix username: `@doordashcon/:matrix.org`
- Polkadot address: `12zsKEDVcHpKEWb99iFt3xrTCQQXZMu477nJQsTBBrof5k2h`
- Current rank: `1`
- Date of initial induction: `2022/09/29`
- Date of last report: `2024/05/04`

> - Area(s) of Expertise/Interest: `Frame & Runtime`

Reviewer: ```suggestion
- Area(s) of Expertise/Interest: `FRAME & Runtime`
```

Grateful that I have been active in adding new features and collaborated with some higher ranked members of the fellowship, their guidiance has been invaluable to me.

### Features
Adding the [Secretary collective](https://github.com/polkadot-fellows/runtimes/pull/347) to the Collective System chain.

### Mainteinance
Refactoring the storage traits, with V1 pallet syntax removed the `generator` traits and method should be moved to `types` directory [[FRAME] Cleanup storage generator types](https://github.com/paritytech/polkadot-sdk/pull/5408).

Members should not be required to manually call `bump` to begin the next salaly cycle, this adds a service task [Task(bump) Pallet Salary](https://github.com/paritytech/polkadot-sdk/pull/5163). 

In the future I aim to take on more design decisions regarding FRAME.

Still a lot to learn, overall I am enjoying my experience working with the Polkadot Ecosystem.

## Voting record
*Provide your voting record in relation to required thresholds for your rank.* 

|  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities | Comments |
|---|---|---|---|---|

> |III|70%   |100%  |I have voted on x out of xx referenda in which I was eligible to vote (i.e xx % voting activity). Out of xx referenda in which members of higher ranks were in complete agreement, I have voted in line with the consensus x times (i.e xx % voting agreement).  |*This is an example.* |

Reviewer: ```suggestion
|III|70%   |100%  |I have voted on x out of xx referenda in which I was eligible to vote (i.e xx % voting activity). Out of xx referenda in which members of higher ranks were in complete agreement, I have voted in line with the consensus x times (i.e xx % voting agreement).  | |
```

I think there should be real number in here?

Author: This is part of the template: https://github.com/polkadot-fellows/Evaluations/blob/main/0000-evidence-template.md#voting-record

Reviewer 2: > I think there should be real number in here?

That row displays the example that comes with the evidence report template.
Jesse is Rank 1, so he shouldn't fill in anything for the rank 3 record.

It is probably best to remove this first row of the voting record table that has the example, @Doordashcon. ✍️

# Evidence-0002: Retention at Rank I

|                 |                                                                                             |
| --------------- | ------------------------------------------------------------------------------------------- |
| **Report Date** | Date of submission (2024/08/21)                                                             |
| **Submitted by**| Jesse Chejieh                                                 |

## Member details

- Matrix username: `@doordashcon/:matrix.org`
- Polkadot address: `12zsKEDVcHpKEWb99iFt3xrTCQQXZMu477nJQsTBBrof5k2h`
- Current rank: `1`
- Date of initial induction: `2022/09/29`
- Date of last report: `2024/05/04`
- Area(s) of Expertise/Interest: `Frame & Runtime`

## Reporting period

- Start date: 2024/05/05
- End date: 2024/08/24

## Evidence

> I believe my contributions over the past period demonstrate my commitment to Polkadot's development and are in accordance with a Rank I member of the fellowship.

Reviewer: ```suggestion
I believe my contributions over the past period demonstrate my commitment to Polkadot's development and are in accordance with a Rank I member of The Fellowship.
```

It's a name in this case.

> I have been actively involve in the development on the polkadot runtime and continued maintainance of the Frame module of the Polkadot-SDK.

Reviewer: ```suggestion
I have been actively involved in the development on the Polkadot runtime and continued maintenance of the FRAME module of the Polkadot-SDK.
```

> Grateful that I have been active in adding new features and collaborated with some higher ranked members of the fellowship, their guidiance has been invaluable to me.

Reviewer: Not sure if I understand the first part of the sentence?

Author: These two mean the same thing.

"I have been actively involve in the development on the polkadot runtime and continued maintainance of the Frame module of the Polkadot-SDK."

"Grateful that I have been active in adding new features and collaborated with some higher ranked members of the fellowship, their guidiance has been invaluable to me."

So one can go.

### Features
Adding the [Secretary collective](https://github.com/polkadot-fellows/runtimes/pull/347) to the Collective System chain.

> ### Mainteinance

Reviewer: Are you active in any communication channels?

Author: I believe I am active in the Polkadot Technical Fellowship Channel on Matrix.

Author: and I am also active in the SE channel on Matrix

Refactoring the storage traits, with V1 pallet syntax removed the `generator` traits and method should be moved to `types` directory [[FRAME] Cleanup storage generator types](https://github.com/paritytech/polkadot-sdk/pull/5408).

Members should not be required to manually call `bump` to begin the next salaly cycle, this adds a service task [Task(bump) Pallet Salary](https://github.com/paritytech/polkadot-sdk/pull/5163). 

> In the future I aim to take on more design decisions regarding Frame.

Reviewer: ```suggestion
In the future I aim to take on more design decisions regarding FRAME.
```

It's a name.

Still a lot to learn, overall I am enjoying my experience working with the Polkadot Ecosystem.

## Voting record
*Provide your voting record in relation to required thresholds for your rank.* 

|  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities | Comments |
|---|---|---|---|---|

> |III|70%   |100%  |I have voted on 0 out of 0 referenda in which I was eligible to vote (100% voting activity) | There are not many things a Rank I can vote on, looking forward to vote in the future |

Reviewer 2: Please make sure you fill in your information based on the requirements for Rank 1. Do not overwrite the example given in the template.

Author: updated

Reviewer 2: This is still incorrect.
You need to fill in the information based on the requirements **for your rank**. Since you are Rank 1, you need to look at the requirements for Rank 1, not Rank 3.
I have indicated the precise section that you need to complete in the image below.
![VotingRecord](https://github.com/user-attachments/assets/13135958-5cc4-4e72-92ee-7b015afbcb40)

FYI: As a Rank 1, you are eligible to vote on all referenda posted here: https://collectives.subsquare.io/fellowship/tracks/1. Please make sure to provide an accurate voting record.

Author: @anaelleltd if I understand correctly during the period from my last submision there where 2 proposal up for vote on https://collectives.subsquare.io/fellowship/tracks/1?

and I am to only update the section you indicated on the image?

Reviewer 2: Correct. You can see an example of what tdimitrov submitted for his retention below.

![image](https://github.com/user-attachments/assets/8daf4cc1-d458-489a-ac4e-b655366a165d)

Thank you!
